owner/src/main/java/org/the_chance/honeymart/ui/util/StringResourceImpl.kt

```diff
@@ -0,0 +1,46 @@
+package org.the_chance.honeymart.ui.util
+
+import android.content.Context
+import org.the_chance.honeymart.domain.util.ErrorHandler
+import org.the_chance.honeymart.domain.util.ValidationState
+import org.the_chance.owner.R
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class StringResourceImpl @Inject constructor(
```

Comment:
Wouldn't it be better to call it `Dictionary `as Bareq mentioned instead of `StringResource`. @TarekIdrees

Reply:
I think it will be more readable to refer to the type of Dictionary to be something like : StringDictionary , What do you think about it ?@Shehab-m